pants

```diff
@@ -126,6 +126,7 @@ done
 no_regen_pex="${NO_REGEN_PEX:-${TRAVIS}}"
 if [[ "${test_goal_used}" == 'true' && "${no_regen_pex}" != 'true' ]]; then
   "$HERE/build-support/bin/bootstrap_pants_pex.sh"
+  echo -e "Finished bootstrapping pants.pex for integration tests.\n" >&2
```

Comment:
Maybe add. "Set the env var `NO_REGEN_PEX=true` to skip this step in the future."